lib/turbograft.rb

```diff
@@ -14,16 +14,20 @@ class Config
     self.controllers = ["ActionController::Base"]
```

Comment:
Maybe we should remove this from here too? We don't need to patch the framework controller since users can add the include in the `ApplicationController`